HISTORY.rst

```diff
@@ -4,9 +4,20 @@ Release History
 ===============
 
 
-unreleased
+0.21.4
 +++++++++++++++
 
+**IoT Hub updates**
+
+* Add command group `az iot hub endpoint update`, which will allow you to update your endpoint without deleting it.
```

Comment:
endpoint -> message-endpoint

Reply:
![image](https://github.com/Azure/azure-iot-cli-extension/assets/28658112/bdaf48aa-ec6c-4689-b35e-cc0fd987226c)

Reply:
I blame the cat